makei/build.py

```diff
@@ -108,6 +108,8 @@ def map_ibmi_json_var(path):
         # if include path is not empty or *NONE then wrap in single quotes
         if len(includePath) > 0 and [v.upper() for v in includePath] != ["*NONE"]:
             incdir='\''+'\' \''.join(includePath)+'\''
+        elif len(includePath) == 1:
+            incdir=includePath[0].upper()
```

Comment:
Do we need to capitalize the incdir only in the elif case? Also in the if case each individual path will have a set of quotes around it as well as a set of quotes to enclose the entire incdir, but in the elif case there will only be the enclosing quotes. Is this expected?

Reply:
Yes, it is only in the ELSE is the *NONE, in the IF clause we have quoted directories and we want to preserve their case.